packages/common/aws.js

```diff
@@ -45,7 +45,7 @@ const improveStackTrace = (fn) =>
       return await fn(...args);
     } catch (err) {
       setErrorStack(err, tracerError.stack);
-      err.message = `${err.message}; Function params: ${args}`;
+      err.message = `${err.message}; Function params: ${JSON.stringify(args, null, 2)}`;
```

Comment:
Oh, I did this, but I'm not sure if we want to spread out the errors across all lines.  Maybe nuke the `, null, 2`?

Reply:
Doesn't the null, 2 just put it in an easier to read format? I think that's fine

Reply:
Yeah, I thought this was just leftover from my debugging, but Lauren added in on purpose so I'm ok with it.